src/dls_python3_skeleton/__main__.py

```diff
@@ -174,6 +184,20 @@ def existing(args):
     )
 
 
+def clean_repo(args):
+    path: Path = args.path
+
+    assert path.is_dir(), f"Expected {path} to be an existing directory"
+
+    branches = [x[2:] for x in str(git("branch", "--list")).split("\n")]
```

Comment:
```suggestion
    branches = [x[2:] for x in str(git("branch", "--format=%(refname:short)", cwd=path)).split("\n")]
```
Although maybe this should be in a `def list_branches(path: Path) -> List[str]` function as it's used in 3 places?

Reply:
Good point :thinking: